src/events.rs

```diff
@@ -122,6 +122,31 @@ macro_rules! make_mouse_event {
     };
 }
 
+macro_rules! make_pointer_event {
+    ($name:ident, $type:literal) => {
+        make_mouse_event!($name, $type => web_sys::PointerEvent);
+
+        impl $name {
+            #[inline] pub fn pointer_id(&self) -> i32 { self.event.pointer_id() }
+
+            #[inline] pub fn width(&self) -> i32 { self.event.width() }
```

Comment:
Although this is called `width` in the DOM, I wonder if that's the best name.

Maybe it should be renamed to `pointer_width`, to make it more clear what it's doing?